### ISSUE TITLE
Grobid "processFulltextDocument" skipping some references 

### ISSUE THREAD

**Issue by almugabo** [kermitt2/grobid#979]
in processing references of some reports (by using **processFulltextDocument**) I noticed that Grobid seems to skip some pages 

for example, when the following file is processed , the references extracted seems to start after the third page 
(it jumps to references starting with the letter B ) 

[FAO_report_biodiversityforFoodAgriculture.pdf](https://github.com/almugabo/ref_scholarly_docs/files/10207228/FAO_report_biodiversityforFoodAgriculture.pdf)

see results 
[xref_raw_test.txt](https://github.com/almugabo/ref_scholarly_docs/files/10207237/xref_raw_test.txt)

it does also not help if one adds few pages in the front : 

[FAO_report_biodiversityforFoodAgriculture_1.pdf](https://github.com/almugabo/ref_scholarly_docs/files/10207227/FAO_report_biodiversityforFoodAgriculture_1.pdf)

see results 
[xref_raw_test_1.txt](https://github.com/almugabo/ref_scholarly_docs/files/10207236/xref_raw_test_1.txt)

any idea on how to deal with this ?

**Comment by karatekaneen** [comment]
You can usually fix this by adding some training data based on similar documents to the segmentation model. We run in to this quite often and continously add more training data. 
You can see more in the [Grobid docs](https://grobid.readthedocs.io/en/latest/Training-the-models-of-Grobid/)

**Comment by almugabo** [comment]
Thanks @karatekaneen . I would like to give it a try. 
A perhaps silly question: 
did I understood it correctly that the steps to follow for this are : 
(1) Put a couple of documents were this happens in a folder (input folder) 
(2) run - in batch mode - the "createTraining" command 
(3) find the "xxxreferenceSegmenter.tei.xml" files and change the "div" to include the missing part ?

**Comment by karatekaneen** [comment]
Yes, you're almost correct. First of all, since a whole page is missing I think it's the `segmentation` model you want to train rather than the `ReferenceSegmenter` model.

And regarding the process the steps are correct but you need to add a couple more:
4. Copy the `xxx.segmentation.tei.xml` as well as the file with only `segmentation` to `grobid-trainer/resources/dataset/segmentation` and put the xml in the `tei` folder and the .segmentation file into the `raw` folder.
5. run the training by `gradlew train_segmentation` 
6. build or run the project, have forgotten how to do this but I think it's just `gradlew run` or `gradlew build`?
